### PR TITLE
Fixes level 1 singularity eating containment field and being able to move

### LINKED ISSUE #87716: When the size of the singularity is 1, it eat's the containment field and move.
### Client Version:

515.1637

### Issue Summary:

When i create a size 1 singularity, it eats containtment fields and can move

### Round ID:

_No response_

### Test Merge Information:

_No response_

### Reproduction Steps:

1. Spawn a size 1 singularity

### PR DESCRIPTION
**Pull request by RengaN02**
## About The Pull Request

Fixes: https://github.com/tgstation/tgstation/issues/87716

Fixes this 

https://github.com/user-attachments/assets/d31c5f97-1576-404b-8427-a4089446fd5f
## Why It's Good For The Game
Its a fix pr
## Changelog
:cl: Rengan
fix: fixes singularity eating containtment field
/:cl:

### CODE REVIEW

**Comment by RengaN02** [comment]
> ok but does that fix it somehow eating the shields

image
yes

File: code/modules/power/singularity/singularity.dm [hunk 1]
```diff
@@ -66,6 +66,7 @@
 	var/datum/component/singularity/new_component = AddComponent(
 		singularity_component_type, \
 		consume_callback = CALLBACK(src, PROC_REF(consume)), \
+		roaming = (move_self && current_size >= STAGE_TWO), \
```
Comment (mc-oofert) [inline code/modules/power/singularity/singularity.dm:69]: this isnt changed later anyway

**Comment by mc-oofert** [comment]
i dont think its supposed to eat the flooring

**Comment by Xkallubar** [comment]
> i dont think its supposed to eat the flooring

It absolutely is supposed to eat the flooring.

**Review by Ghommie — APPROVED** [review]

**Pull request merged**